src/components/notification-badge/notification-badge.yaml

```diff
@@ -1,7 +1,7 @@
 examples:
 - name: default
   data:
-    text: 3
+    badgeText: 3
 - name: with-text
   data:
-    text: New
+    badgeText: New
```

Comment:
Is this related to the account header? What's the reason for breaking the `text` convention govuk are using?